src/runtime/disco/nccl/nccl.cc

```diff
@@ -162,7 +162,7 @@ struct CCLThreadLocalContext {
 
 void InitCCL(Session sess, IntTuple device_ids) {
   DRef func = sess->GetGlobalFunc("runtime.disco." TVM_DISCO_CCL_NAME ".init_ccl_per_worker");
-  LOG(INFO) << "Initializing " TVM_DISCO_CCL_NAME " with devices: " << device_ids;
+  // LOG(INFO) << "Initializing " TVM_DISCO_CCL_NAME " with devices: " << device_ids;
```

Comment:
Let’s recover this line

Reply:
Oh i got what you mean. Do you want to remove verbose logging message?

Reply:
Yup, that was my intent, as correct behavior should generally be silent.  That said, this is useful for debugging, so I've replaced it with `DLOG(INFO)`.